integrations/ultralytics/README.md

```diff
@@ -0,0 +1,93 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# SparseML-ultralytics/yolov5 integration
+This directory provides a SparseML integrated training script for the popular
+[ultralytics/yolov5](https://github.com/ultralytics/yolov5)
+repository.
+
+Using this integration, you will be able to apply SparseML optimizations
+to the powerful training flows provided in the yolov5 repository.
+
+Some of the tasks you can perform using this integration include, but are not limited to:
+* model pruning
+* quantization-aware-training
+* sparse quantization-aware-training
+* sparse transfer learning
+
+## Installation
+To use both the script, clone both repositories, install their dependencies,
+and copy the integrated training script into the yolov5 directory to run from.
+
+```bash
+# clone
+git clone https://github.com/ultralytics/yolov5.git
+git clone https://github.com/neuralmagic/sparseml.git
+
+# copy script
+cp sparseml/integrations/ultralytics/main.py yolov5
+cd yolov5
+
+# install dependencies
+pip install -r requirements.txt
+pip install sparseml
+```
+
+
+## Script
+`integrations/ultralytics/main.py` modifies
```

Comment:
One more small change that I missed, can we rename this file to train.py? This way there is a one to one mapping without a lookup of names for the file(s) in the integration repo to the files stored in sparseml